css/src/components/textfield.css

```diff
@@ -0,0 +1,5 @@
+.Textfield--helptext {
+  display: flex;
+  justify-content: space-between;
+  height: 20px;
```

Comment:
There is no token for `20px` please check its usage once

Reply:
need to check this designer as we are using helptext in this and in helptext we have a div wrap in it with mt-3 which is causing a problem in this so due to this we need to set the height 20px extenally.